profiles/all.yaml

```diff
@@ -0,0 +1,7 @@
+# This file was auto generated and is intended for debugging an entire build, take caution if editing manually.
```

Comment:
The cmake process cannot detect `offload_vio.device_tx` and the other three plugins correctly. It only reports that `offload_vio` is not enabled.

Reply:
This branch was working with an outdated plugins.yaml file. I just committed an update to fix it.